app/assets/stylesheets/container_topology.css

```diff
@@ -19,16 +20,33 @@ kubernetes-topology-graph {
 
 .checkbox-inline {
     font-size: 16px;
-    padding-left: 32px;
+    padding-right: 10px;
 }
 
-.refresh {
+.fa-refresh {
```

Comment:
This is almost certainly wrong, you're changing `fa-refresh` for the whole application. Use another class name here, please.